edkahara/iReporter
# [finishes #162406750] create a user signup api endpoint

__What does this PR do?__
Create a user signup api endpoint

__Description of tasks to be completed.__
Create an api endpoint that facilitates the creation of a new user. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-user-signup-api-endpoint-#162406750 branch. Start the environment using env\Scripts\activate. Run flask run on the terminal and go to Postman. To perform the tests run nosetest.

__Relevant pivotal tracker stories.__
#162406750
